### PR TITLE
Include babel-preset-env documentation

### PR DESCRIPTION
**Pull request by xtuc**
Fixes #1081.

### CODE REVIEW

File: _sass/pages/_docs.scss [hunk 1]
```diff
@@ -36,4 +36,8 @@
     padding: 0;
     border-left: none;
   }
+
+  img {
+    width: 100%;
+  }
```
Comment (xtuc) [inline _sass/pages/_docs.scss]: The picture was overflowing outside the container: 
image

**Comment by xtuc** [comment]
We'll need to make some change to the babel-preset-env README (avoid inline code blocks for example).

File: _sass/pages/_docs.scss (hunk 1 above)
Comment (raspo) [inline _sass/pages/_docs.scss]: `max-width` is usually preferred when resizing images for "responsiveness"

File: _sass/pages/_docs.scss [hunk 2]
```diff
@@ -36,4 +36,8 @@
     padding: 0;
     border-left: none;
   }
+
+  img {
+    max-width: 100%;
```
Comment (xtuc) [inline _sass/pages/_docs.scss]: @raspo Thanks, I changed it.

**Comment by existentialism** [comment]
@xtuc I tweaked the GitHub/Edit this Link logic to allow us to point to a non monorepo

**Comment by existentialism** [comment]
We should wait for https://github.com/babel/babel-preset-env/pull/96 to get merged, as it removes a $ and converts some relative links.

**Comment by hzoo** [comment]
merged

**Comment by xtuc** [comment]
@existentialism Nice job, thanks. Links are working correctly.

The logic is becoming quite complex but isn't visible by contributors.

LGTM

**Pull request merged**